bulldozer/config_fetcher.go

```diff
@@ -189,8 +189,9 @@ func (cf *ConfigFetcher) unmarshalConfigV0(bytes []byte) (*Config, error) {
 				Trigger: Signals{
 					Labels: []string{"merge when ready", "merge-when-ready", "merge_when_ready"},
 				},
-				DeleteAfterMerge: configv0.DeleteAfterMerge,
-				Method:           configv0.Strategy,
+				DeleteAfterMerge:       configv0.DeleteAfterMerge,
+				AllowMergeWithNoChecks: false,
```

Comment:
I think it's fine to set these explicitly, but it's not required since the zero value of the field is the desired default value.

Reply:
👍🏻  I was on the fence about that, but figured it might be better to be explicit about the setting.